sys/dev/etherswitch/e6000sw/e6000sw.c

```diff
@@ -217,6 +217,7 @@ e6000sw_probe(device_t dev)
 	phandle_t switch_node;
 #else
 	int is_6190;
+	int is_6190x;
```

Comment:
So, for is_6190 the code assumes one of the two properties is present (otherwise this is left uninitialised). That's a bit dodgy, but for is_6190x you definitely can't assume that otherwise you will break existing hints that predate the existence of is_6190x.